### PR TITLE
Added alert option for the docs:lint script

### PR DESCRIPTION
**Pull request by przemyslaw-zan**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Internal: Added `-a`/`--alert` option for the `docs:lint` script that allows choosing alert level for Vale. Valid values: `error`, `warning`, `suggestion`.

---

### Additional information

Requires:
- https://github.com/ckeditor/ckeditor5/pull/15307

### CODE REVIEW

**Review by psmyrek — APPROVED** [review]
LGTM 🎉

**Comment by psmyrek** [comment]
Shouldn't this PR target the `stable` branch?

File: scripts/vale/vale.js [hunk 1]
```diff
@@ -40,16 +46,19 @@ const defaultPatterns = [
...
 	boolean: [
 		'save'
 	],
 	alias: {
+		a: 'alert',
 		d: 'directory',
 		s: 'save'
 	},
 	default: {
+		alert: 'warning',
```
Comment (AnnaTomanek) [inline scripts/vale/vale.js:61]: The default should be the same as set in the `.vale.ini` file in the project. Why do you override the global project setting here?

Comment (przemyslaw-zan) [reply scripts/vale/vale.js:61]: Created new issue for this: https://github.com/ckeditor/ckeditor5/issues/15382

**Pull request merged**